wallet-api/src/Wallet/Emulator/Types.hs

```diff
@@ -86,9 +90,10 @@ import           Wallet.UTXO                (Address', Block, Blockchain, Height
 import qualified Wallet.UTXO.Index          as Index
 
 -- agents/wallets
-newtype Wallet = Wallet { getWallet :: Int }
+newtype Wallet = Wallet Int
```

Comment:
Why?

Reply:
Solving a problem that was actually much better fixed by deriving our JSON instances `via newtype`.

Reverted.